changelogs/gitlab.md

```diff
@@ -2,6 +2,10 @@
 
 You can find full Gitlab Release changelog [here](https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CHANGELOG.md).
 
+## 10.2.2
+- https://about.gitlab.com/2017/11/28/gitlab-10-2-2-released/
+- https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CHANGELOG.md
```

Comment:
Change to : https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CHANGELOG.md#1022-2017-11-23